test/config_test/examples/cifar10-pytorch.test.yml

```diff
@@ -3,6 +3,7 @@ experimentName: default_test
 maxExecDuration: 15m
 maxTrialNum: 2
 trialConcurrency: 2
+nniManagerIp: 52.168.146.89
```

Comment:
Please do not write nniManagerIp in *.test.yml,  write it in training_service.yml

Reply:
remvoed from example.yml